Makefile

```diff
@@ -316,9 +316,8 @@ define stringreplace
 	$(build_depsbindir)/stringreplace $$(strings -t x - $1 | grep '$2' | awk '{print $$1;}') '$3' 255 "$(call cygpath_w,$1)"
 endef
 
-install: $(build_depsbindir)/stringreplace
+install: $(build_depsbindir)/stringreplace $(BUILDROOT)/doc/_build/html
```

Comment:
we should try to ensure somehow that this is up to date, so we don't use a stale copy